client-flink/flink-1.14-shaded/pom.xml

```diff
@@ -67,6 +71,7 @@
               <include>com.google.guava:guava</include>
               <include>io.netty:*</include>
               <include>org.apache.commons:commons-lang3</include>
+              <include>org.roaringbitmap</include>
```

Comment:
does it work w/o `:*`?

Reply:
yes it's equal to org.roaringbitmap:*，and i am going to specify its artifactId.